lib/home/view/tasks_page.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:flutter/material.dart';
 import 'package:tasks_repository/tasks_repository.dart';
 import 'package:to_do_list_kamilg/home/home.dart';
-
+import 'package:app_ui/src/dialogs/bottom_animated_dialog.dart';
```

Comment:
the only import you should add is 
`import 'package:app_ui/app_ui.dart';`
Probably you forgot to add dialogs export to `app_ui` library.
Please go to `app_ui.dart` (library file) and check the exports there, probably
`export './dialogs/dialogs.dart'` is missing